src/components/navbar/Header.js

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import BasicMenu from "./DropDownMenu";
+import { makeStyles } from "@material-ui/core/styles";
+import { Logo } from "../../utils/assets";
+import { createTheme } from "@mui/material/styles";
+const useStyles = makeStyles({
+  navbarButtonsMargin: {
+    marginLeft: "25px",
+  },
+});
+const theme = createTheme({
+  typography: {
+    variant: "h6",
+    component: "div",
+  },
+});
+export default function Navbar() {
+  const classes = useStyles();
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar
+        style={{
+          backgroundColor: "#261f18",
```

Comment:
make a default color folder and store this color there as [CONSTANT](https://www.robinwieruch.de/javascript-naming-conventions/#:~:text=javascript%20naming%20conventions%3A%20constant):
export const ZEUS = "#261f18"

if you want use this website to know the color name: https://chir.ag/projects/name-that-color/#261F18
or give it a custom name as you like since we don't have a designer